apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.tsx

```diff
@@ -58,12 +58,11 @@ const LoginError: React.FC<{ status?: number }> = ({ status }) => {
 };
 
 const LoginComponent: React.FC<LoginProps> = ({ className }) => {
-  const navigate = useNavigate();
   const location = useLocation();
   const from = (location.state as { from?: Location })?.from?.pathname || '/';
   const authCallback = useCallback(() => {
-    navigate(from, { replace: true });
-  }, [from, navigate]);
+    window.location.replace(`/dashboard${from}`);
```

Comment:
can this get a query param for `?next_page` which is the Django default via `django.contrib.auth.views.LoginView`? maybe something like:

```
const [ searchParams, setSearchParams ] = useSearchParams();
const next_page? = searchParams.get("next_page");

...
window.location.replace(`${next_page ?? `/dashboard${from}``);
```